.phpcs.xml

```diff
@@ -1,12 +1,7 @@
 <?xml version="1.0"?>
-<ruleset name="WikibaseCodeSniffer">
-	<rule ref="Wikibase" />
-
-	<rule ref="PSR1.Classes.ClassDeclaration.MissingNamespace">
-		<exclude-pattern>Wikibase/Sniffs/</exclude-pattern>
-	</rule>
-	<rule ref="Squiz.Classes.ValidClassName.NotCamelCaps">
-		<exclude-pattern>Wikibase/Sniffs/</exclude-pattern>
+<ruleset>
+	<rule ref="Wikibase">
+		<exclude name="MediaWiki.Commenting.MissingCovers" />
```

Comment:
Is this still necessary even though the Wikibase rule set also disables that sniff?

Reply:
Good catch, I missed this when playing around with this patch.

Reply:
Yeah, I saw the WIP version yesterday :)